src/rospack.cpp

```diff
@@ -1643,7 +1453,7 @@ Rosstackage::computeDepsInternal(Stackage* stackage, bool ignore_errors, const s
     }
     else if(!stackages_.count(dep_pkgname))
     {
-      if (stackage->is_wet_package_ && (ignore_missing || isSysPackage(dep_pkgname)))
+      if (stackage->is_wet_package_ && ignore_missing)
```

Comment:
This change is the main one that creeps me out. As I say, all of this has AFAICT worked fine for us for months, but I don't really understand the implications of dropping the `isSysPackage` check here.

Reply:
Without checking for `isSysPackage` I would expect this to flag a system dependency as missing if `ignore_missing` isn't specified. I don't know if `computeDepsInternal` is used anywhere but I would assume that this change breaks the function.

Reply:
In our hound instance (which covers all of ros-desktop-full), the only hits I have on `computeDepsInternal` and `computeDeps` are from within this repo. Same with the `Rosstackage`, `Rosstack`, and `Rospack` keywords.

So perhaps even more of this is vestigial than I thought.

Reply:
It looks like the only place that even includes `rospack.h` is here:

https://github.com/ros/ros/blob/1.14.7/core/roslib/src/package.cpp#L29

But the only thing it instantiates with it is the backcompat `ROSPack` object, for its `run` method that invokes a Python rospack subprocess.

Reply:
Usage of this function isn't my concern here. I think removing the `isSysPackage` condition breaks the behavior / logic of this function. If there is no alternative I would rather expect the function needs to be removed.

Reply:
@mikepurvis Friendly ping.